test/test_mission_manager/test_mission_manager.cpp

```diff
@@ -3,27 +3,36 @@
 #include <unity.h>
```

Comment:
I am not sure what these changes are but probably remove any changes to the test for mission manager. If you want to commit them that is fine because they will be changed for the new mission modes anyway.